Update giganto-client to `6fe7654`

sophie-cluml opened the pull request:
There are 4 major commits.

The first commit 83aff2fd66134f649134770e6de20e2b3dc28ab9 uses `i64` for time data and removes `end_time`.  It will:

Close: #1329
Close: #1333

The second commit b1f9256ca9382a14f0bfc6b6eb1dfd49c05c7828 brings bincode back to v1. It will:

Close: #1320

The third commit 7aa5923a602b3814e5d0e233ebd662dbe2cc3fba adds time format related integration tests for GraphQL APIs, which is a prep for the chrono-to-jiff migration. And the fourth commit 5a16ebd0f57a81f1de9bce5d594a90ede019c4ff adds tests for storage key builder functions, which also serves as a prep for the chrono-to-jiff migration. However, even aside from the chrono-to-jiff effort, this work is valuable on its own because it adds missing unit tests for the storage key builders. The third and fourth commits will:

Close: #1322

All in all, this PR will:

Close: #1335

kimhanbeom reviewed (changes requested):
1. It seems the current changelog could be revised to better highlight the changes relative to the previous release (v0.25.1).
    - Text to be removed
       - Changed `last_time` field to `end_time` for raw events.....
       - Changed `duration` field to `end_time` for Conn raw events in....
    - Text to be added
       - Removed the `last_time` field from all network raw event structs. The combination of `start_time` and `duration` sufficiently represents the event’s time range, making `last_time` redundant.`
    - Text to be modified (replace `end_time` to `last_time`)
       - The `start_time` field represents the session start time and complements
        the existing `end_time` field.
2. In the src/lib.rs derive usage examples, could you please also remove the `end_time` field and update the type of the `start_time` field.

sophie-cluml commented:
@kimhanbeom Thank you for the review.

- I removed and added the CHANGELOG as you suggested. However, regarding your suggestion on the modification, I chose to remove the phrase ```and complements the existing `end_time` field``` instead. I believe ```The `start_time` field represents the session start time``` already explains the meaning of `start_time` sufficiently on its own.
- Regarding your comment on `lib.rs`: I don’t think the `ConnRawEvent` struct in the documentation example must match  the struct that Giganto defines in production code. The purpose of the example is to provide helpful reference information for anyone looking to understand `lib.rs` (mainly with various field types). Since we already have an `i64` field in the example, I believe removing `end_time` makes the example simple and concise. I removed it, and since I believe this is unrelated to the actual removal of the `end_time` field in production code, I separated the removal into its own  commit f3de6eb. And to avoid any misunderstanding that the documentation example struct must match the production struct, I also adjusted the example struct name.

NOTE: With the latest force-push, only the CHANGELOG of the first commit was adjusted and the last commit was newly added.

zmrdltl reviewed (approved):
Looks good—revert back to bincode v1 is clean and consistent
